service-proxy.js

```diff
@@ -172,6 +172,24 @@ function rateLimit(req, buildRes) {
     var isExitNode = self.isExitFor(req.serviceName);
     if (isExitNode) {
         self.rateLimiter.createServiceCounter(req.serviceName);
+        self.rateLimiter.createKillSwitchCounter(req.serviceName);
```

Comment:
lets punt on "should be at ingress" until we have rolled out the favor incoming and drop outgoing socket in all languages.